### PR TITLE
LPS-201062

### PR DESCRIPTION
**Pull request by thallescezar**
- LPS-201062 Fixing pagination when listing object entries
- LPS-201062 Removing unused method
- LPS-201062 Format Source

### CODE REVIEW

**Comment by thallescezar** [comment]
ci:test:objects

**Comment by thallescezar** [comment]
ci:test:relevant

**Comment by victorhcunha** [comment]
ci:test:object

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/item/selector/ObjectEntryItemSelectorViewDescriptor.java [hunk 1]
```diff
@@ -139,10 +139,21 @@ public SearchContainer<ObjectEntry> getSearchContainer()
...
+
 				searchContainer.setResultsAndTotal(
-					_getObjectEntries(
-						searchContainer.getCur(), searchContainer.getDelta(),
-						ParamUtil.getString(_portletRequest, "keywords")));
+					() -> TransformUtil.transform(
+						page.getItems(),
+						objectEntry -> ObjectEntryUtil.toObjectEntry(
+							_objectDefinition.getObjectDefinitionId(), objectEntry)),
+					Math.toIntExact(page.getTotalCount()));
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/item/selector/ObjectEntryItemSelectorViewDescriptor.java:160]: Hey @thallescezar , can you use `GetterUtil.getInteger(page.getTotalCount());`? Like this [example](https://github.com/liferay/liferay-portal/blob/cc7034192fede4a5dee2cad027c7a650e87456fe/modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectFieldResourceTest.java#L135). Thanks!

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by thallescezar** [comment]
ci:test:sf

**Comment by thallescezar** [comment]
ci:test:objects

**Comment by thallescezar** [comment]
ci:test:relevant

**Comment by thallescezar** [comment]
ci:test:object

**Review by guilhermedcamacho — APPROVED** [review]

**Comment by guilhermedcamacho** [comment]
ci:forward:force

**Pull request closed without merging**